### ISSUE TITLE
Multiple ways to navigate site

### ISSUE THREAD

**Issue by thejkayway** [thejkayway/m4apdx#15]
[SC 2.4.5 Multiple Ways](https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html)

We should have at least 2 of the following:

- [ ] Site search
- [ ] Site map
- [ ] Table of contents
- [ ] List of related pages
- [x] List of all available web pages

Our nav items count as a list of all available web pages.

A site search or sitemap would likely be the most functional of the above options, with a site search being a more differentiated way of navigating than the existing nav items (and thus preferable for fulfilling SC 2.4.5).

**Comment by thejkayway** [comment]
Gatsby docs on adding search to a static site: https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-search/.

- Algolia
  - Pros
    - free tier of 10k searches / month
    - fast to implement (30 min to an hour)
    - a simple/minimal solution, aka easy to yank out later if we decide to do something else
  - Cons
    - limit of 10k searches / month unless we pay
    - may be slightly slower than build-time indexing<sup>1</sup>
    - no "i wrote this myself" cred to brag about at the water cooler
- Build-Time Search
  - Pros
    - no limit to the number of searches per month
    - may be faster than build-time indexing<sup>1</sup>
  -  Cons
     - Longer development time
     - Higher code complexity
     - Longer builds -> longer time between deploy and seeing changes

<sup>1</sup> don't yet understand algolia's cache invalidation behavior compared to the netlify CDN, so hard to say if there will be a noticeable performance difference. i expect any difference to be negligible
